lib/cisco_node_utils/client/nxapi/client.rb

```diff
@@ -186,6 +186,7 @@ def req(type, command_or_list)
     debug("Sending HTTP request to NX-API at #{@http.address}:\n" \
           "#{request.to_hash}\n#{request.body}")
     begin
+      @http.use_ssl = false
```

Comment:
Added comment.  Unfortunately, the node I was seeing the error on (n3k-52) is now down, so I can't verify the error is fixed.  Regardless, explicitly setting `use_ssl` to `false` looks like it will prevent any EOFErrors from happening